scripts/utils/bcdi_center_of_rotation_ID01.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# BCDI: tools for pre(post)-processing Bragg coherent X-ray diffraction imaging data
+#   (c) 07/2017-06/2019 : CNRS UMR 7344 IM2NP
+#   (c) 07/2019-present : DESY PHOTON SCIENCE
+#       authors:
+#         Jerome Carnis, carnis_jerome@yahoo.fr
+# 		  input from Marie-Ingrid Richard, mrichard@esrf.fr
+
+from math import tan, pi
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+import sys
+
+helptext = """
+Calculate the vertical correction and correction along the beam to apply to a
+nanocrystal to move it at the center of rotation along the beam direction.
+At two eta angles, perform a piy-scan of the particle. 
```

Comment:
Please remove the trailing whitespace.
```suggestion
At two eta angles, perform a piy-scan of the particle.
```